### PR TITLE
Popup-windows pull

### PR DESCRIPTION
**Pull request by Ramaunspoken**
Changed the required changes.
Added a Popup window to the work section to give it a good style with help from Javascript.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -516,6 +511,110 @@ a:disabled:hover {
...
+  justify-content: center;
+}
+
+.article-modal {
+  border-radius: 20px;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  width: 100%;
+  height: 100%;
```
Comment (omarsalem7) [inline style.css:535]: - [x] It will be better if you remove this property to make your design match the one in Figma.

File: style.css [hunk 2]
```diff
@@ -516,6 +511,110 @@ a:disabled:hover {
...
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  line-height: 20px;
+  font-weight: 500;
+  max-width: 146px;
+  margin: 10px;
+}
```
Comment (omarsalem7) [inline style.css:597]: - [x] Kindly make sure to design modal buttons like the one in Figma.

| Your design  | Figma design|
| ------------- | ------------- |
| image  | image |

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Ramaunspoken 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- Correct GitHub follow ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Ramaunspoken) [reply style.css:535]: Thanks for the insights @omarsalem7

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅

## Hi @Ramaunspoken 👋

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**



## Highlights:
* Well written PR description ✔️ 
* All tests passed ✔️ 


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**